.circleci/config.yml

```
@@ -113,13 +113,14 @@ jobs:
           name: Collect artifacts when an exception occurs
           # language=sh
           command: |
-            mkdir -p /exception-artifacts
-            for f in $(ls -d /tmp/ide-probe/screenshots/*/) ; do cd  $f/artifacts ; zip -r "/exception-artifacts/$(basename ${f}).zip" . ; done
+            for f in $(ls -d ~/artifacts/*/) ; do cd  $f ; zip -r "../$(basename ${f}).zip" . ; done
+            mkdir -p ~/artifacts/zip
+            find ~/artifacts -name '*.zip' | xargs cp -t ~/artifacts/zip
```

Nit: actually, `-t` is pretty obscure/uncommon... pls use the long option here for clarity
```suggestion
            find ~/artifacts -name '*.zip' | xargs cp --target-directory ~/artifacts/zip
```